webapp/components/Comment.vue

```diff
@@ -28,13 +27,27 @@
       </no-ssr>
       <!-- eslint-disable vue/no-v-html -->
       <!-- TODO: replace editor content with tiptap render view -->
+
+      <div v-if="isCollapsed" v-html="comment.contentExcerpt" style="padding-left: 40px;" />
+      <div
+        v-show="comment.content !== comment.contentExcerpt"
+        style="text-align: right;  margin-right: 20px; margin-top: -12px;"
+      >
+        <a v-if="isCollapsed" style="padding-left: 40px;" @click="isCollapsed = !isCollapsed">
+          {{ $t('comment.show.more') }}
+        </a>
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-06-28T05:55:49Z" title="Friday, June 28th 2019, 7:55:49 am +02:00">Jun 28, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -23,10 +23,27 @@
     <!-- eslint-disable vue/no-v-html -->
     <!-- TODO: replace editor content with tiptap render view -->
     <ds-space margin-bottom="small" />
-    <div style="padding-left: 40px;" v-html="comment.contentExcerpt" />
-    <!-- eslint-enable vue/no-v-html -->
+    <div v-if="!isActive" v-html="comment.contentExcerpt" style="padding-left: 40px;" />
+    <div v-show="comment.content !== comment.contentExcerpt">
+      <span
+        v-if="!isActive"
+        style="padding-left: 40px; color: #17b53f; cursor:pointer"
+        @click="isActive = !isActive"
+      >
+        {{ $t('comment.show.more') }}
+      </span>
+    </div>
+    <div v-if="isActive" v-html="comment.content" style="padding-left: 40px;" />
+    <span
+      v-if="isActive"
+      @click="isActive = !isActive"
+      style="padding-left: 40px; color: #17b53f; cursor:pointer"
```


I would imagine, if there is a way, use the color from the style guide as color name via CSS?
I think I have seen this at other places …